[NON-MODULAR] Adds pronouns to Ethereals, as well as underwear

Mega-Drones opened the pull request:
## About The Pull Request

Gives Ethereals the option to chose their pronouns instead of being forced into non-binary, also allows Ethereals to wear underwear.

## How This Contributes To The Skyrat Roleplay Experience

Ethereals should be able to chose their pronouns and underwear for the personal preference of their players

## Changelog

:cl:
qol: Gives Ethereals the option to chose a their pronouns instead of being forced into non-binary, also allows them to wear underwear if they wish to do so
/:cl:

optimumtact commented:
massive lore violation and against the wishes of the original developer

thestubborn commented:
i dont really have anything else to say since oranges nailed it on the head

respect the wishes of the original creator, dude

Mega-Drones commented:
I am not forcing you guys to give your Ethereals a gender, it just allows the option for those who want it. My char will continue to be considered gendered wether or not this PR goes through, as that is my own personal preference; this is just a small change to do stuff like "X snaps _his/her_ fingers" instead of "X snaps their fingers" for the sake of more emersion-
once again, for those who want it. 
This is by no means a forced change on all Ethereals, those who want to keep their Ethereals genderless can still do so

GoldenAlpharex reviewed (commented):
They've still got the `AGENDER`  species_traits so I'm not sure how this one'll work out.

Mega-Drones commented:
Removed ` AGENDER` as I wouldn't want it to come and bite us in the future, even though it doesn't effect the gender text ingame during testing.
Also removed `NO_UNDERWEAR`, when attempting to give an Ethereal underwear it would cause the character selection window to bluescreen

Yawet330 commented:
> massive lore violation and against the wishes of the original developer

don't care + we don't use your lore most of the time

Yawet330 commented:
Unironically though;

Furry ERP will be furry ERP. It literally has not stopped Ethereals from roleplaying as a gender thusfar. Oranges is (once more) shitposting on the Skyrat Repo in order to stir shit in PR comments.

He keeps getting away with it too.

Anyway yeah we don't use /tg/ lore for the most parts so kekw on that.

hugmeorelse commented:
> massive lore violation and against the wishes of the original developer

We don't follow your lore. Also, didn't tg have issues with following the wishes of original developers themselves? I remember quite the kerfuffle over that.

hugmeorelse commented:
> massive lore violation and against the wishes of the original developer

We don't follow your lore. Also, didn't tg have issues with following the wishes of original developers themselves? I remember quite the kerfuffle over that.

Either ways, I'll ask someone about a Skyrat ethereal if this is really that big of an issue to you.

hugmeorelse commented:
> massive lore violation and against the wishes of the original developer

We don't follow your lore. I would argue we follow it as much as you do. 😄

Also, didn't tg have issues with following the wishes of original developers themselves? I remember quite the kerfuffle over that. 😛

Either ways, I'll ask someone about a new and separate Skyrat ethereal if this is really that big of an issue to ya'll.

Yawet330 commented:
> 

It doesnt actually give them genitalia as far as I am aware (and can tell from the code). This just allows them to pick pronouns.

Mega-Drones commented:
^
This PR is a QOL change, which is what it is tagged with. 
I'll edit the description to better portray this

Gandalf2k15 commented:
After careful consideration, we have decided that this is not something we'd want
